config/src/main/java/org/axonframework/config/EventProcessingConfigurer.java

```diff
@@ -260,6 +260,8 @@ EventProcessingConfigurer registerTokenStore(String processorName,
      */
     EventProcessingConfigurer usingPooledStreamingEventProcessors();
 
+    EventProcessingConfigurer usingEventProcessorBuilder(EventProcessorBuilder processorBuilder);
```

Comment:
Curious: I agree it's good to have this method. But, what did we need it for as part of the multi-tenancy support?